Casks/f/follow.rb

```diff
@@ -0,0 +1,37 @@
+cask "follow" do
+  version "0.0.1,21"
+  sha256 "eeddce6afcdfa37e1688fb0328b8562a47f81407e4cd6922a7b0ae2591f72460"
+
+  url "https://github.com/RSSNext/Follow/releases/download/v#{version.csv.first}-alpha.#{version.csv.second}/Follow-#{version.csv.first}-alpha.#{version.csv.second}-macos-universal.dmg",
+      verified: "github.com/RSSNext/Follow/"
+  name "Follow"
+  desc "Next generation information browser"
```

Comment:
```suggestion
  desc "Information browser"
```

We should omit the marketing jargon (see [documentation](https://docs.brew.sh/Cask-Cookbook#dos-and-donts)).